lua/yagpdbcc.lua

```diff
@@ -45,35 +45,6 @@ end
 ---@param callback fun(response: lsp.CompletionResponse|nil)
 function source:complete(params, callback)
     callback({
-		{ label = 'true' },
-		{ label = 'false' },
-		{ label = 'if' },
-		{ label = 'else' },
-		{ label = 'range' },
-		{ label = 'with' },
-		{ label = 'define' },
-		{ label = 'template' },
-		{ label = 'block' },
-		{ label = 'not' },
-		{ label = 'and' },
-		{ label = 'or' },
-		{ label = 'ne' },
-		{ label = 'eq' },
-		{ label = 'lt' },
-		{ label = 'le' },
-		{ label = 'gt' },
-		{ label = 'ge' },
-		{ label = 'add' },
-		{ label = 'sub' },
-		{ label = 'mult' },
-		{ label = 'div' },
-		{ label = 'fdiv' },
-		{ label = 'log' },
-		{ label = 'mod' },
-		{ label = 'pow' },
-		{ label = 'sqrt' },
-		{ label = 'nil' },
-		{ label = 'end' },
```

Comment:
Did we mean to remove these? If not, the changes to `gen_code_completion.py` might not be working quite as expected.

Reply:
It's actually the changes made to `syntax/yagpdbcc.vim` which are causing this... I'll move them where they belong.